### PR TITLE
More misbehaving metastable states

### PR DESCRIPTION
**Pull request by scopatz**
Pm-136 to start

### CODE REVIEW

File: src/atomicgen.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```
Comment (FlanFlanagan) [inline src/atomicgen.py]: Is this correct?

Comment (scopatz) [reply src/atomicgen.py]: Yes, you are never supposed to run from system python directly.  This runs with the user's python, which may be system one, but doesn't have to be.

**Comment by FlanFlanagan** [comment]
======================================================================
FAIL: test_data.test_metastable_id
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/robert/pyne/tests/test_data.py", line 200, in test_metastable_id
    assert_equal(data.metastable_id(310720000, 1), 310720002)
AssertionError: None != 310720002

Getting this error. Does this require the decay PR as well?

**Comment by scopatz** [comment]
You need to do a `nuc_data_make --clean 1`, I think

**Comment by scopatz** [comment]
I think that this is ready to go

**Comment by scopatz** [comment]
After I look at fixing the tests...

**Comment by scopatz** [comment]
This is superceded by #934

**Comment by scopatz** [comment]
Actually, I was wrong, this needs to go in too. CC @FlanFlanagan @asmeurer @makeclean

**Pull request merged**